docker/docker-compose-dev.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.9"
+
+services:
+  cubeseed:
+    container_name: cubeseed-backend-dev
+    image: cubeseed
+    build:
+      dockerfile: docker/Dockerfile.local
+      context: ..
+      args:
+        - DJANGO_SETTINGS_MODULE=cubeseed.settings
+    environment:
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PASS=${DB_PASS}
+      - DB_HOST=db
+    command: manage.py runserver 0.0.0.0:8000
```

Comment:
can you add a `.env.example` file so contributors can know what to include in their env files when running the docker container

Reply:
Testing the auth token endpoint on localhost also gave cors error You might want to look into that

![cors error](https://github.com/Cubeseed/Backend/assets/35562131/b6a3daa1-3ddc-471f-8284-78752638b1a7)

Reply:
Added `docker/.env` with the basics (enough for development).